tests/AppFixture.js

```diff
@@ -1,2 +1,3 @@
-baseline = /\d+ variables et \d+ paramètres référencés/
+baseline = /OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies./
```

Comment:
Increase test resilience and readability by matching only a significant subset of the text.